### PR TITLE
use the black config from conda-smithy

### PR DESCRIPTION
**Pull request by CJ-Wright**
This PR has only formatting content.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 79
```
Comment (beckermr) [inline pyproject.toml]: Let’s put this at 88 but otherwise LGTM.

**Comment by CJ-Wright** [comment]
@beckermr ready for another round.

**Comment by beckermr** [comment]
LGTM!

If you want to add flake8 too I’m game but no rush.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**